frontend/components/OrgPage/EditBox.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function EditBox() {
+  return (
+    <div>
+      <p>Edit</p>
+    </div>
+  );
```

Comment:
Make sure you have prettier set up
```suggestion
  )
```